homeassistant/components/zwave_js/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Z-Wave JS",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/zwave_js",
-  "requirements": ["zwave-js-server-python==0.31.1"],
+  "requirements": ["zwave-js-server-python==0.31.2"],
```

Comment:
We already merged this bump, right? We should rebase here on latest dev. :+1:

Reply:
good point, thanks!